api.h

```diff
@@ -0,0 +1,6 @@
+#ifndef API_H
+#define API_H
+
+bool useAbility(int abilityID);
```

Comment:
This is a public API function, so it should have documentation explaining what it is, what it does and what the parameters are. For example, start with something like:
```suggestion
/**
 * Tries to use the given ability.
 * 
 * Returns false if the ability cannot be used.
 */
bool useAbility(int abilityID);
```

Reply:
See https://google.github.io/styleguide/cppguide.html#Function_Comments for some good guidelines.